publications/index.html

```diff
@@ -6,13 +6,39 @@ <h1>How to cite BioSchemas</h1>
 To cite BioSchemas please use the website URL, https://bioschemas.org and "Gray, A.J.G, Goble, C.A. and Jimenez, R., 2017. Bioschemas: From Potato Salad to Protein Annotation. In International Semantic Web Conference (Posters, Demos & Industry Tracks)."
 
 <h1>Publications and Talks</h1>
-<div style="float:right">
+<div style="float:right;padding-left:25px">
   <iframe src="https://www.slideshare.net/bioschemas/slideshelf" width="615" height="470" frameborder="0" allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>
 </div>
 <div id="pub_list">
   <p>A list of publications and talks in which Bioschemas was presented:</p>
   <ol>
-    <li>ELIXIR All Hands, 17-20 June 2019, Lisbon, Portugal
+    <li>ELIXIR All Hands Meeting 2020, 9-11 June 2020, Amsterdam, the Netherlands
```

Comment:
But it will be virtual now. Should the original place still be listed? If this is how it appears on their web site, it is ok, otherwise I suggest update it

Reply:
Fixed in commit 1fe9bf1